### ISSUE TITLE
[BUG][PYTHON] Generated files have simple syntax errors from unescaped quotes

### ISSUE THREAD

**Issue by syntaxaire** [OpenAPITools/openapi-generator#13373]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?

##### Description

The Python files produced by this OpenAPI spec have syntax errors that make them impossible to run without editing.

##### openapi-generator version

Latest

##### OpenAPI declaration file content or url

Minimal reproduction:
https://gist.github.com/syntaxaire/6469bbbc7f866918eea0143343f38638

##### Generation Details

`docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli:latest generate --global-property skipFormModel=false -i /local/openapi.json -g python -o /local/openapi-client`

##### Steps to reproduce

1. Download the minimal reproduction from the gist link.
2. Run the command in "Generation Details".  (`${PWD}` is a PowerShell construction, substitute your own working directory if necessary.)
3. Inspect `openapi-client/openapi_client/api/sample_api.py`
4. Line 66 has a syntax error:
```python
                'allowed_values': {
                    ('close',): {

                        "&quot;&gt;DATE&quot;,": "">DATE","
                    },
                },
```
There are extra quotation marks that make the file unparseable. In the OpenAPI spec, they were escaped but here they are not.

**Comment by spacether** [comment]
How about sing the 6.2.0 python generator or if your version is before that then it is the python-experimental generator.
In the test spec there is an [enum with escaped characters](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/unit_test_spec/3_0_3_unit_test_spec.yaml#L3314)
Which generates [this model](https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/3_0_3_unit_test/python/unit_test_api/model/enum_with_escaped_characters.py#L38) which has [these passing tests](https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/3_0_3_unit_test/python/test/test_models/test_enum_with_escaped_characters.py)

**Comment by syntaxaire** [comment]
This seems to be fixed in at least v6.2.1.
